packages/manager/apps/pci-gateway/public/translations/common/Messages_fr_FR.json

```diff
@@ -1,8 +1,6 @@
 {
   "pci_projects_project_public_gateway_title": "Gateway",
-  "pci_projects_project_public_gateways_intro_part_1": "Les Gateways fournissent une connectivité aux instances privées, permettant d’accéder de manière sécurisée aux réseaux externes ou d’exposer l'application à un accès externe :",
-  "pci_projects_project_public_gateways_intro_part_2": "Dans le premier cas, Gateway agit comme une passerelle SNAT, autorisant les instances à atteindre Internet, mais pas le contraire.",
-  "pci_projects_project_public_gateways_intro_part_3": "Dans le second exemple, il s’agit de donner la possibilité d’exposer un service derrière une instance privée ou un Load Balancer à Internet en utilisant des Floating IP.",
+  "pci_projects_project_public_gateways_intro": "Les Gateways fournissent une connectivité aux instances privées, permettant d’accéder de manière sécurisée aux réseaux externes ou d’exposer l'application à un accès externe :",
```

Comment:
as you removed the list, I think you don't need the `:` anymore

Reply:
Oh yes good point. And I needed to change the text a little too.